libnavui-maneuver/src/test/java/com/mapbox/navigation/ui/maneuver/RoadShieldContentManagerTest.kt

```diff
@@ -0,0 +1,754 @@
+                        }
+                    }
+                )
+            }
+            every { secondary } returns mockk {
+                every { id } returns "secondary_3"
+                every { componentList } returns listOf(
+                    mockk {
+                        every { node } returns mockk<RoadShieldComponentNode> {
+                            every { shieldUrl } returns "https://shield.mapbox.com/secondary_3"
+                        }
+                    }
+                )
+            }
+            every { sub } returns mockk {
+                every { id } returns "sub_3"
+                every { componentList } returns listOf(
+                    mockk {
+                        every { node } returns mockk<RoadShieldComponentNode> {
+                            every { shieldUrl } returns "https://shield.mapbox.com/sub_3"
+                        }
+                    }
+                )
+            }
+        },
+        mockk {
+            every { primary } returns mockk {
+                every { id } returns "primary_4"
+                every { componentList } returns listOf(
+                    mockk {
+                        every { node } returns mockk<RoadShieldComponentNode> {
+                            every { shieldUrl } returns null
+                        }
+                    }
+                )
+            }
+            every { secondary } returns null
+            every { sub } returns mockk {
+                every { id } returns "sub_4"
+                every { componentList } returns listOf(
+                    mockk {
+                        every { node } returns mockk<RoadShieldComponentNode> {
+                            every { shieldUrl } returns null
+                        }
+                    }
+                )
+            }
+        },
+        mockk {
+            every { primary } returns mockk {
+                every { id } returns "primary_5"
+                every { componentList } returns listOf(
+                    mockk {
+                        every { node } returns mockk<RoadShieldComponentNode> {
+                            // same as primary_0 intentionally
+                            every { shieldUrl } returns "https://shield.mapbox.com/primary_0"
+                        }
+                    }
+                )
+            }
+            every { secondary } returns null
+            every { sub } returns null
+        }
+    )
+
+    @Before
+    fun setup() {
+        mockkObject(RoadShieldDownloader)
+    }
+
+    @Test
+    fun `empty lists returned if indices empty`() = coroutineRule.runBlockingTest {
+        val expected = RoadShieldResult(
+            shields = emptyMap(),
+            errors = emptyMap(),
+        )
+
+        var actual: RoadShieldResult? = null
+        pauseDispatcher {
+            actual = manager.getShields(testManeuvers, emptySet())
+        }
+
+        Assert.assertEquals(expected, actual)
```

Comment:
NIT

```suggestion
        assertEquals(expected, actual)
```

Same thing applies to the rest of the tests 👇